### PR TITLE
Add Stun tracking to Character sheets.

### PR DESCRIPTION
**Pull request by JFranklin3000**
Added a field by health to track stun.

### CODE REVIEW

**Review by guswadsworth12 — CHANGES_REQUESTED** [review]
Add this field to NPC sheet as well.

**Comment by JFranklin3000** [comment]
added to the NPC sheet.

File: templates/actor/actor-npc-sheet.hbs [hunk 1]
```diff
@@ -22,6 +22,10 @@
           {{!-- Health --}}
           {{> "systems/essence20/templates/actor/parts/actor-health.hbs"}}
         </div>
+        <div class="npc-health">
+          {{!-- Health --}}
```
Comment (phildominguez) [inline templates/actor/actor-npc-sheet.hbs:26]:
```suggestion
          {{!-- Stun --}}
```

File: templates/actor/parts/actor-common.hbs [hunk 2]
```diff
@@ -9,4 +9,7 @@
 
   {{!-- Health --}}
   {{> "systems/essence20/templates/actor/parts/actor-health.hbs"}}
+
+    {{!-- Stun --}}
```
Comment (phildominguez) [inline templates/actor/parts/actor-common.hbs:13]:
```suggestion
  {{!-- Stun --}}
```

File: templates/actor/parts/actor-stun.hbs [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<div class="stats-container flexrow flex-group-center" style="border-color: {{system.color}};">
+  <div class="flexcol flex-group-center">
+    <label>{{localize 'E20.ActorStun'}}</label>
+    <div class="flexrow" style="align-items: center;">
+      <input class="centered-input" type="number" name="system.stun.value" value="{{system.stun.value}}" />
+      <span>/</span>
+      <input class="centered-input" type="number" name="system.stun.max" value="{{system.health.value}}" />
```
Comment (phildominguez) [inline templates/actor/parts/actor-stun.hbs:7]: Thoughts on making this input disabled?

**Review by phildominguez — CHANGES_REQUESTED** [review]
Works nicely! Small stuff.

File: templates/actor/parts/actor-stun.hbs (hunk 3 above)
Comment (JFranklin3000) [reply templates/actor/parts/actor-stun.hbs]: I like it and added it.

File: templates/actor/parts/actor-common.hbs (hunk 2 above)
Comment (JFranklin3000) [reply templates/actor/parts/actor-common.hbs]: fixed

File: templates/actor/actor-npc-sheet.hbs (hunk 1 above)
Comment (JFranklin3000) [reply templates/actor/actor-npc-sheet.hbs]: fixed

File: templates/actor/parts/actor-stun.hbs (hunk 3 above)
Comment (JFranklin3000) [reply templates/actor/parts/actor-stun.hbs]: Yea that looks better.

**Review by phildominguez — APPROVED** [review]

**Review by guswadsworth12 — APPROVED** [review]
Looks and works good now man

**Pull request merged**